index.js

```diff
@@ -0,0 +1,87 @@
+module.exports = {
+  parser: "@typescript-eslint/parser",
+
+  extends: ["plugin:react/recommended", "prettier/@typescript-eslint", "plugin:prettier/recommended", "prettier/react"],
+
+  plugins: ["simple-import-sort", "jest", "@typescript-eslint", "react-hooks", "react", "prettier", "import"],
+
+  ignorePatterns: ["coverage", "**/dist", ".cache-loader", "node_modules", ".yalc"],
+
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: "module",
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+
+  settings: {
+    react: {
+      version: "detect",
+    },
+  },
+
+  env: {
+    browser: true,
+    node: true,
+    jest: true,
+    es6: true,
+  },
+
+  rules: {
+    "prettier/prettier": "error",
+
+    // Base
+
+    "no-unused-expressions": ["error", { allowTernary: true }],
+    "no-console": "off",
+    "no-confusing-arrow": "off",
+    "no-else-return": "off",
+    "no-return-assign": ["error", "except-parens"],
+    "no-underscore-dangle": "off",
+    camelcase: "off",
+    "prefer-arrow-callback": ["error", { allowNamedFunctions: true }],
+    "class-methods-use-this": "off",
+    "no-restricted-syntax": "off",
+    "no-param-reassign": ["error", { props: false }],
+    "arrow-body-style": "off",
+
+    // Imports
+
+    "simple-import-sort/sort": "error",
+    "sort-imports": "off",
+    "import/order": "off",
+    "import/no-extraneous-dependencies": "off",
+    "import/first": "error",
+    "import/newline-after-import": "error",
+    "import/no-duplicates": "error",
+
+    // Typescript
+
+    "@typescript-eslint/no-unused-vars": [
+      "warn",
+      {
+        argsIgnorePattern: "^_",
+        varsIgnorePattern: "^_",
+        args: "after-used",
+        ignoreRestSiblings: true,
+      },
+    ],
+
+    // React
+
+    "react-hooks/rules-of-hooks": "error",
+    "react-hooks/exhaustive-deps": "error",
+    "react/prop-types": "off",
+    "react/no-multi-comp": "off",
+    "react/jsx-filename-extension": "off",
+    "react/no-unescaped-entities": "off",
+    "react/destructuring-assignment": "off",
+    "react/display-name": "off",
+
+    // Jest
+
+    "jest/no-focused-tests": "error",
+    "jest/no-identical-title": "error",
```

Comment:
```suggestion
    "jest/no-identical-title": "error",
    "jest/no-truthy-false": "error",
    "jest/consistent-test-it": "error"
```
These 2 rules I added suggest here seem like a valuable addition.